doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,107 @@
+     > You can explicitly match a parameter with a Feed even the names don't match by decorating the parameter with the `FeedParameter` attribute:  
+     >
+     > ```csharp
+     > public IFeed<string> Message { get; }
+     > 
+     > public async ValueTask Share([FeedParameter(nameof(Message))] string msg)
+     > {
+     > }
+     > ```
+
+     > [!TIP]  
+     > You can also opt in or out of implicit matching of Feeds and command parameters by the decorating the current assembly or class with the `ImplicitFeedCommandParameters` attribute:  
+     >
+     > ```csharp
+     > [assembly:ImplicitFeedCommandParameter(false)]
+     >
+     > [ImplicitFeedCommandParameter(true)]
+     > public partial record MyModel
+     > ```
+
+1. Using factory methods
+
+    Commands can also built manually using the static class [`Command`](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Presentation/Commands/Command.cs).  
+    This class provides factory methods for creating commands.
+
+    - Async
+
+    ```csharp
+    public ICommand MyCommand => Command.Async(async(ct) => await PingServer(ct));
+    ```
+
+    > [!Note]  
+    > You need not worry about the `=>` operator and that the `Command.Async` will be called over and over, this value is cached and will only be called once.  
+    The benefit of using `=>` in place of traditional `{ get; } = ...`, is that local methods are accessible (otherwise these would have been required to be intialized in the constructor).
+
+    - `Create` & `Create<T>`;
+
+    To create a command you can use the fluent API of `ICommandBuilder` provided in the `Command.Create` factory methods, which provides the following three methods:
+
+    - Given - This method takes a Feed (or a State!) and configures a command which will be triggered whenever a new value is available to the Feed.
+
+        ```csharp
+        public IFeed<int> PageCount => ...
+
+        public IAsyncCommand MyCommand => Command.Create(builder => builder.Given(PageCount));
+        ```
+
+    - When - Limits the command execution to a set prerequisite - in other words sets the 'can execute' of the command.
+
+        ```csharp
+        public IAsyncCommand MyCommand => Command.Create<int>(builder => builder.When(i => i > 10));
+        ```
+
+    - Then - Sets the actual asynchronous callback to be invoked when the Command is executed. This method will be genereric if there's a preceding parameter setting (via Given or When).
+
+        ```csharp
+        public IAsyncCommand MyCommand => Command.Create(builder => builder.Then(async ct => await ExecuteMyCommand(ct)));
+        ```
+
+    Here's a complete example:
+
+    ```csharp
+    public IAsyncCommand MyCommand => 
+        Command.Create(builder => 
+            builder
+            .Given(CurrentPage)
+            .When(currentPage => currentPage > 0)
+            .Then(async (currentPage, ct) => await NavigateToPage(currentPage, ct)));
+    ```
+
```

Comment:
Can you expand on why/when you should use factory instead of auto generation of command?

Reply:
@dr1rrb I would need your help on this.

Reply:
I've changed the description on line 138 to clarify that the recommended way is code generation.

1. Using factory methods

    Adding Commands via code generation is sufficient enough to probably cover all scenarios. However sometimes you'd want to fine-grain your Commands and declare them in an explicit manner.  
    Commands can be built manually using the static class [`Command`](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Presentation/Commands/Command.cs).  
    This class provides factory methods for creating commands.

Reply:
@dr1rrb it's encouraging that u say "yet", it means that you're planning to implement it 🍻💥